lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -195,6 +197,9 @@ class InputConfiguration {
 
   /// Whether to hide the text being edited.
   final bool obscureText;
+
+  /// Whether to  to enable autocorrection. Only available on Webkit.
+  final bool autocorrect;
```

Comment:
It would be good to define somewhere what's the difference is between "autocorrect", "autocomplete", and "spell checking". I was confused initially, and I may be confused again 2 months from now :)

Reply:
:) sure I can add it. Just to clarify, I am not adding anything special to what is written on developer.mozilla.org

In my observation, inside our code, developers so far didn't add documentations for the cases documented in MDN. Current documentation covers things which is different in Flutter, or documented cases we needed to added a workaround to fix a known issue, or documented the browser differences which might not be obvious to the reader.

Reply:
I agree with Nurhan that we shouldn't duplicate all the info about `autocorrect`, `autocomplete`, etc. Since it's already explained on MDN, would it be better to just add a link to it?

Reply:
I added link to MDN. Thanks for the review again.